### ISSUE TITLE
Question on initializing batch_apn_path file

### ISSUE THREAD

**Issue by nickrobinson** [frankgwo/FaceComparing-PyTorch#1]
Hi @frankgwo. I ran across your code yesterday and ran across an issue when trying to reproduce your results. I realized I do not know how to seed the initial version of the batch_apn_path file. How did you choose initial triplets to add to that file?

**Comment by frankgwo** [comment]
Hi. My batch_apn_path file is a txt which stores all the possible triplets in the batch, generated by the "MiniBatchAllTriplets". In the txt, every line makes up a triplet, containing paths of anchor, positive and negative pics.
Btw,  i have to friendly remind u that i m a beginner in python/dl, and my codes may be quite dirty and inefficient. 
Hope it might help. ;)

**Comment by nickrobinson** [comment]
Thanks for the quick reply! I see what you are saying now. For the moment I am generating my triplet file based on labels I have for my training set with no focus on generating hard or semi-hard triplets. Once I get that working I will try out the MiniBatchAllTriplets approach.
